servers/zts/src/main/java/com/yahoo/athenz/zts/ZTSImpl.java

```diff
@@ -2678,7 +2700,7 @@ public Response postInstanceRegisterInformation(ResourceContext ctx, InstanceReg
             // able to validate the certificate during refresh operations
 
             if (insertX509CertRecord(ctx, cn, provider, certReqInstanceId, certSerial,
-                    InstanceProvider.ZTS_CERT_USAGE_CLIENT.equalsIgnoreCase(certUsage)) == null) {
+                    InstanceProvider.ZTS_CERT_USAGE_CLIENT.equalsIgnoreCase(certUsage), certExpiryTime, null) == null) {
```

Comment:
we cannot use the certExpiryTime here since the cert signer may not honor what the server is asking  and instead use another expiry time. So the correct value here is to extract the expiry from the newCert that we're about send back to the client.

also, for the last hostname argument, why are we passing null instead of info.getHostname()?